src/containers/ProjectDashboardContainer.jsx

```diff
@@ -43,9 +44,11 @@ class ProjectDashboardContainer extends Component {
   }
 
   handleSelect({ option }) {
+    const { actions } = this.props;
     this.setState({
       option,
     });
+    actions.createTranslation('project_contents', option.value);
```

Comment:
Here you go: [`selectTranslation(original, translations, language)`](/zooniverse/pandora/pull/19/files#diff-26d3d8bb803fd9662fecef23eb6074fbR114)